keyboards/miniaxe/readme.md

```diff
@@ -0,0 +1,16 @@
+MiniAxe
+===
+
+![MiniAxe](https://i.imgur.com/1ApzrCz.jpg)
+
+Yet another split ortholinear keyboard with 3x5+3 keys.
+
+Keyboard Maintainer: [ka2hiro](https://github.com/ka2hiro) [@ka2hiro](https://twitter.com/ka2hiro) 
+Hardware Supported: MiniAxe PCB, ATMEGA32U4
```

Comment:
This line needs two spaces at the end for the file to be rendered as intended.

```suggestion
Hardware Supported: MiniAxe PCB, ATMEGA32U4  
```

Reply:
Added two spaces. Thanks.